src/js/view/controls/templates/menu/menu.js

```diff
@@ -0,0 +1,11 @@
+export const MenuTemplate = (isSubmenu, name) => {
```

Comment:
Just a question, it seems like this MenuTemplate creates a settings menu or settings submenu. Are they interchangeable or are they a bit separate, because it seemed like SettingsMenu was more so a specific type of Menu?

Reply:
They're almost interchangeable after this refactor. But because a lot of people have custom implementations of our product I wanted to avoid changes the HTML class structure in any way that could affect custom skins/plugins.